dspace-oai/pom.xml

```diff
@@ -159,6 +159,10 @@
                      <groupId>org.apache.commons</groupId>
                      <artifactId>commons-lang3</artifactId>
                 </exclusion>
+                <exclusion>
```

Comment:
I'm a bit worried about this exclusion as it is an artifact from an old, different layout, jackson version. The mapper class are now in the jackson-databinder artifact so if we really use this staff we could hit a classnotfound exception at runtime somewhere... BTW the jtwig-spring version in our depency is quite old the new one is: http://jtwig.org/